Explorer/Assets/Scripts/ECS/StreamableLoading/AssetBundles/LoadAssetBundleSystem.cs

```diff
@@ -32,7 +33,7 @@ public partial class LoadAssetBundleSystem : LoadSystemBase<AssetBundleData, Get
         private readonly AssetBundleManifest assetBundleManifest;
 
         internal LoadAssetBundleSystem(World world, IStreamableCache<AssetBundleData, GetAssetBundleIntention> cache,
-            AssetBundleManifest assetBundleManifest, MutexSync mutexSync) : base(world, cache, mutexSync)
+            AssetBundleManifest assetBundleManifest, MutexSync mutexSync, ConcurrentLoadingBudgetProvider loadingBudgetProvider) : base(world, cache, mutexSync, loadingBudgetProvider)
```

Comment:
You should pass an interface everywhere instead of the implementation, please change it everywhere